ui/ts/pages/stores.ts

```diff
@@ -138,7 +141,24 @@ module AdminViews {
      * StorePage show the details of a single node.
      */
     export module StorePage {
+      import NavigationBar = Components.NavigationBar;
+
       class Controller {
+        private static defaultTargets: NavigationBar.Target[] = [
+          {
+            title: "Overview",
+            route: ""
+          },
+          {
+            title: "Graphs",
+            route: "graph"
+          }
+        ];
+
+        private static isActive: (targ: NavigationBar.Target) => boolean = (t: NavigationBar.Target) => {
```

Comment:
Well it made sense to me, my thought was that a single nav bar was going to switch between discrete options that are at roughly the same "level" of the page - the top navigation bar determines the base route, while the subpage navigation bar just determines a single parameter of the route.

It made sense to me.